native_client/BUILD

```diff
@@ -111,7 +119,10 @@ tf_cc_shared_object(
     ] + if_cuda([
         "//tensorflow/core:core",
     ]),
-    includes = ["c_speech_features", "kiss_fft130"] + DECODER_INCLUDES,
+    includes = ["c_speech_features", "kiss_fft130"] + select({
```

Comment:
Maybe it'd be easier to read if we keep `DECODER_INCLUDES` and we move the select when creating the variable ?

Reply:
I tried to do it in the same declaration but I missed a  `}` so I decided to move them into two declarations then concatenate to `DECODER_INCLUDES` and DECODER_SOURCES. What do you guys think @reuben